### PR TITLE
react-addons-pure-render-mixin to pureComponent

### PR DESCRIPTION
**Pull request by ZeroCho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,15 +78,14 @@
...
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2",
     "rimraf": "^2.5.1",
     "standard-version": "^2.2.1",
     "tomchentw-npm-dev": "^4.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.6 || ^15.0.0",
-    "react-addons-pure-render-mixin": "^0.14.0 || ^15.0.0"
+    "react": "^15.3.X",
```
Comment (tomchentw) [inline package.json]: Could you switch to `^15.3.0`? Thanks!

File: lib/PrismCode.js [hunk 2]
```diff
@@ -74,7 +62,7 @@ var PrismCode = function (_Component) {
   }]);
 
   return PrismCode;
-}(_react.Component);
+}(_react.PureComponent);
```
Comment (tomchentw) [inline lib/PrismCode.js]: Please revert the changes in the `lib` folder.

**Comment by tomchentw** [comment]
Ref #21

**Comment by ZeroCho** [comment]
@tomchentw 
Revert all changes in lib folder?
Okay i will do that tomorrow. Its midnight here.

**Comment by tomchentw** [comment]
@ZeroCho yes, everything in `lib` folder. We should change only `src`. Thanks!

**Comment by tomchentw** [comment]
@ZeroCho any chances you can fix this today? Thanks

**Comment by ZeroCho** [comment]
@tomchentw I made mistakes in this PR, so I created new PR

**Comment by tomchentw** [comment]
Released [v4.0.0](https://github.com/tomchentw/react-prism/releases/tag/v4.0.0)
